pontoon/contributors/static/css/contributor.css

```diff
@@ -33,57 +33,46 @@ a.avatar:hover .desc ~ img {
 #username {
   color: #EBEBEB;
   font-size: 48px;
-  letter-spacing: -2px;
-  text-transform: none;
-}
-
-#username .uneditable {
   letter-spacing: normal;
   padding: 8px 0 20px;
+  text-transform: none;
 }
 
-#username .container {
-  display: inline-block;
+#profile-form {
+  display: block;
   position: relative;
+  text-align: left;
+  margin: 0 auto 70px auto;
+  width: 620px;
+}
+#profile-form .field {
+  margin-top: 20px;
+  text-align: left;
 }
 
-#username .container input {
-  background: transparent;
-  border-bottom: 1px dashed transparent;
+#profile-form .field input {
   color: #FFFFFF;
-  font-size: 48px;
-  font-weight: 300;
-  padding: 4px 50px;
-  text-align: center;
-  width: 100%;
+  background: #333941;
+  border: 1px solid #4d5967;
+  border-radius: 3px;
+  float: none;
+  width: 290px;
+  padding: 4px;
 
   -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
 
-#username .container input:hover,
-#username .container input:focus {
-  border-color: #888888;
-}
-
-#username .container .submit {
-  bottom: -8px;
-  color: #888888;
-  opacity: 0;
-  padding: 20px 0;
-  position: absolute;
-  right: 0;
-  /* Needed or click doesn't trigger */
-  transition: all 0.1s ease-out;
+#profile-form button {
+  margin-top: 10px;
 }
-
-#username .container input:focus + .submit {
-  cursor: pointer;
-  opacity: 1;
+#email-warning {
+  display: none;
+  margin-top: 5px;
+  color: red;
 }
-
-#username .container input:focus + .submit:hover {
-  color: #7BC876;
+#id_email:focus + span{
+  display: block;
```

Comment:
`contributor.css` is shared between the /settings and /profile page. All the newly introduced CSS rules should be moved to the settings.css file, because they don't have any impact on the /profile page.